flask_resty/__init__.py

```diff
@@ -44,4 +44,4 @@
     pass
 
 
-__version__ = "4.0.0"
+__version__ = "4.0.1"
```

Comment:
patch update should be OK for this diff I think?
https://github.com/4Catalyzer/flask-resty/compare/v4.0.0...master